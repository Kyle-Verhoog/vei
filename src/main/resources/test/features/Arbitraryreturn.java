public class Arbitraryreturn {
  public Arbitraryreturn() {}
  public int m(int x) {
    if (x==42)
      return 87;
    else
      return 17;
  }
}