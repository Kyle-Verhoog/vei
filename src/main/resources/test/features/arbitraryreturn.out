PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
INT ┃ int
IDENTIFIER ┃ m
( ┃ (
INT ┃ int
IDENTIFIER ┃ x
) ┃ )
{ ┃ {
IF ┃ if
( ┃ (
IDENTIFIER ┃ x
== ┃ ==
INTEGER_LITERAL ┃ 42
) ┃ )
RETURN ┃ return
INTEGER_LITERAL ┃ 87
; ┃ ;
ELSE ┃ else
RETURN ┃ return
INTEGER_LITERAL ┃ 17
; ┃ ;
} ┃ }
} ┃ }